packages/cojson/src/coValues/account.ts

```diff
@@ -103,6 +104,36 @@ export class RawControlledAccount<Meta extends AccountMeta = AccountMeta>
     return this.core.node.createGroup(uniqueness);
   }
 
+  /**
+   * Creates a new `CoPlainText` within this Account, with the specified specialized
+   * `CoPlainText` type `T` and optional static metadata.
+   *
+   * @category 3. Value creation
+   */
+  createPlainText<T extends RawCoPlainText>(
+    init?: string,
+    meta?: T["headerMeta"],
+    initPrivacy: "trusting" | "private" = "private",
+  ): T {
+    const text = this.core.node
+      .createCoValue({
+        type: "coplaintext",
+        ruleset: {
+          type: "ownedByGroup",
+          group: this.id,
+        },
+        meta: meta || null,
+        ...this.core.crypto.createdNowUnique(),
+      })
+      .getCurrentContent() as T;
+
+    if (init) {
+      text.insertAfter(0, init, initPrivacy);
+    }
+
+    return text;
+  }
```

Comment:
The create methods are defined on `group` 

Since account extends group we don't need to add this method here because it should be inherited